### PR TITLE
fixed indentation, removed thumbnail size setting

### PR DESCRIPTION
**Pull request by 81jumpstr**
To all reviewing this repo for the first time, "feat/branches" are a bit of a mess. From here on, I'll be naming them "fix/branches"...getting back to standard practice...

### CODE REVIEW

**Comment by royscheeren** [comment]
What is the mess?

File: functions.php [hunk 1]
```diff
@@ -5,43 +5,37 @@ function Setupshop_script_enqueue() {
 
   }
   add_action('wp_enqueue_scripts', 'Setupshop_script_enqueue');
```
Comment (royscheeren) [inline functions.php]: Don't use capital letters in function names. Thats for classes.

File: functions.php [hunk 2]
```diff
@@ -5,43 +5,37 @@ function Setupshop_script_enqueue() {
...
+function arphabet_widgets_init() {
+  register_sidebar( array(
+  	'name'          => 'Filters',
+  	'id'            => 'filter',
+  	'before_widget' => '<div>',
+  	'after_widget'  => '</div>',
+  	'before_title'  => '<h2 class="rounded">',
+  	'after_title'   => '</h2>', ) );
+}
   add_action( 'widgets_init', 'arphabet_widgets_init' );
```
Comment (royscheeren) [inline functions.php]: Consider your function names. Think about what it is you're copy pasting. The `arphabet` is nothing more than a prefix. You shouldn't just take that with. Just make sure the second part of the add_action is the function you're calling.

File: functions.php [hunk 3]
```diff
@@ -5,43 +5,37 @@ function Setupshop_script_enqueue() {
...
 ?>
 
 <?php
 function register_my_menu() {
   register_nav_menu('header-menu',__( 'Header Menu' ));
 }
-add_action( 'init', 'register_my_menu' );
+  add_action( 'init', 'register_my_menu' );
```
Comment (royscheeren) [inline functions.php]: There should be no indentation here.

**Review by royscheeren — CHANGES_REQUESTED** [review]

File: functions.php (hunk 1 above)
Comment (81jumpstr) [reply functions.php]: Ok, no caps...the mess was the way I wasn't branching properly.

**Pull request merged**